src/content/docs/style-guide/writing-docs/article-templates/intro-branch-install-doc.mdx

```diff
@@ -0,0 +1,138 @@
+
+Here are our implementations of the branched install with summaries of the choices made for them. We recommend picking one that matches the approach you want to take and adjust it as needed. Please try to keep this list up-to-date as we add new uses. 
+
+* Vulnerability management: [Import dependabot data](/install/vm/). [See a labeled image of this YAML file](#simple-yaml-example). Details: 
+  * A very simple implementation. A single option selection box, which lets you choose between two files. 
+  * No default-displayed content. 
+  * Note that this is so simple, the tab component might have been a better choice.  
+* [MySQL integration install](/install/mysql). Details: 
+  * A simple implementation with only a single option selection box. 
+  * It takes a default-content-displayed approach. 
+* [Java install](/install/java). Details:  
+  * It takes a [default-content approach](#default-content-decision). This is because there's one install path that most customers take. 
+  * It has only two option selection sections: depending on what's selected in the first option, a second option box is displayed. 
+  * It uses the `recommendedGuided` component to display the guided install (contained in the `appInfo-2` file).
+  * It uses the config file component. 
+* [.NET install](/install/dotnet). [See a labeled image of this yaml file](#complex-yaml-example). Details: 
+  * This doesn't display any default content. It relies entirely on the [`isConditionalStep: true`](#default-content-decision) approach. This is because there's quite a few paths and no one path the majority of customers take (although theoretically we could have chosen one).
+  * This has only two option selection sections: depending on what's selected in the first option, a second option box is displayed. 
+  * It uses the `recommendedGuided` component (contained in the `appInfo-2` file). 
+  * It does not use the config file component: this is because most customers edit the configuration via environment variables or other options and not through direct use of the file. 
+* [Python install](/install/python). Details: 
+  * It does not display default content. It relies entirely on the [`isConditionalStep: true` approach](#default-content-decision). This is because there is no one major path. 
+  * This has quite a few option selection boxes: up to four, depending on what gets selected. 
+  * It does not use the config file component. 
+  * It makes use of comments, which helps understanding of the code and improved maintenance.
+* [Network performance monitoring endpoints](/install/npm). Details: 
+  * A pretty simple implementation. Three option selection boxes that control which of several tables are shown. A default-displayed file with replace overrides that determine which table to show. 
+
+## YAML file examples [#yaml-examples]
+
+The YAML files contain all the logic about how the branched install will work. We have two examples below. 
```

Comment:
I'd review all mentions of YAML, yaml, and .yml and choose which one you want to use for this doc.